pygeoapi/api/__init__.py

```diff
@@ -80,6 +80,7 @@
 
 CHARSET = ['utf-8']
 F_JSON = 'json'
+F_COVERAGEJSON = 'json'
```

Comment:
Why is this not in `FORMAT_TYPES`?

Reply:
Adding to `FORMAT_TYPES` will overwrite F_JSON (same key) in the subsequent dict.  We can address in a subsequent PR.